src/main/resources/application.properties

```diff
@@ -0,0 +1,34 @@
+#cors
+spring_cors_origin=*
```

Comment:
Andrà sovrascritto da un elenco di domini.

Reply:
Certo, possiamo avere già l'elenco dei domini?

Reply:
variano al variare degli ambienti dev, svil, coll, ..... arriveranno ad un parametro del cloudformation. 
Tale parametro è una lista di domini separata da virgole, accettiamo solo connessioni in https

Reply:
ok